src/wovn_helper.php

```diff
@@ -3,11 +3,29 @@
 require_once(__DIR__ . '/wovnio/wovnphp/SSI.php');
 use Wovnio\Wovnphp\SSI;
 
+function remove_dots_from_path($path) {
+  # Removes '/./ in paths and resolves '/../' in paths
+  $path = str_replace('//', '/', $path);
+
+  $path_parts = explode('/', $path);
+  $tmp_out = array();
+  foreach($path_parts as $part){
+    if ($part == '.') {
+      continue;
+    }
+    if ($part == '..') {
+      array_pop($tmp_out);
+      continue;
+    }
+    $tmp_out[] = $part;
+  }
+  return implode('/', $tmp_out);
+}
+
 function wovn_helper_detect_paths($base_dir, $path_of_url, $files) {
   $request_path = $base_dir . $path_of_url;
-  $local_path = realpath($request_path);
-  $inside_base_dir = $local_path && strpos($local_path, $base_dir) === 0;
-  $local_path = $inside_base_dir ? $local_path : $base_dir;
+  $local_path = remove_dots_from_path($request_path);
```

Comment:
Do you think `realpath` function can apply in this case, instead of making our own `remove_dots_from_path` function?

Reply:
Oh hmmmm...maybe the bug is in the behavior after realpath.  Let me try something.

Reply:
I think that it's need prevent directory traversal.
For example, if user request path is `../../../../../../etc/passwd`, we don't permit return `/etc/passwd`.
But `realpath` does not consider such as security side and `remove_dots_from_path` is too.